src/malli/swagger.cljc

```diff
@@ -67,3 +68,102 @@
                                  ::json-schema/definitions definitions
                                  ::json-schema/transform -transform})]
      (cond-> (-transform ?schema options) (seq @definitions) (assoc :definitions @definitions)))))
+
+(defn remove-empty-keys
+  [m]
+  (into (empty m) (filter (comp not nil? val) m)))
+
+(defmulti extract-parameter (fn [in _] in))
+
+(defmethod extract-parameter :body [_ schema]
+  (let [swagger-schema (transform schema {:in :body, :type :parameter})]
+    [{:in          "body"
+      :name        (:title swagger-schema "body")
+      :description (:description swagger-schema "")
+      :required    (not= :maybe (m/type schema))
+      :schema      swagger-schema}]))
+
+(defmethod extract-parameter :default [in schema]
+  (let [{:keys [properties required definitions]} (transform schema {:in in, :type :parameter})]
+    (println "\nextract-parameter definitions:"
+             (with-out-str (clojure.pprint/pprint definitions)))
```

Comment:
these printlns need to go before mergint this

Reply:
Sorry about that (and the commented-out fn above). Meant to clean those up before issuing the PR.